apps/frontend/src/app/3_organisms/EmailNotificationSettingsDialog/EmailNotificationSettingsDialog.tsx

```diff
@@ -274,11 +278,17 @@ const EmailNotificationSettingsDialogComponent: React.FC<
     );
 
     handleUserDataResponse(promise, true);
+
+    if (isOnboarding) {
```

Comment:
If you call it like that, you will never see the last part of the flow (confirmation after save). `onClose` should be called only after you click "Continue".